platform/ios/app/MBXAnnotationView.m

```diff
@@ -12,7 +12,7 @@ - (void)layoutSubviews {
     [super layoutSubviews];
     if (!self.centerView) {
         self.backgroundColor = [UIColor blueColor];
-        self.centerView = [[UIView alloc] initWithFrame:CGRectInset(self.bounds, 5.0, 5.0)];
+        self.centerView = [[UIView alloc] initWithFrame:CGRectInset(self.bounds, 1.0, 1.0)];
```

Comment:
MBXAnnotationView is also used for dropped pins, but this is kind of small for that demo. Let’s stop using an annotation view for dropped pins, so we get better visibility on GL annotations, which are still supported. MBXAnnotationView would continue to be used for other demos like “Add _n_ Point Annotations”.